### DISCUSSION TITLE
Please make it work  without wifi network

### DISCUSSION THREAD

**Discussion by prudhvidas** [localsend/localsend#1423] [Discussion (English, International)]
Is it possible to make this to work without connecting to wifi nerwork

**Comment by Max9199** [comment]
Yes please; would be useful to have work through a mobile hotspot between devices

**Comment by prudhvidas** [comment]
Thank you it works for me

**Comment by LoveMyLand** [comment]
It would be great if it could work via Bluetooth as well.
Many times when I am traveling I want to move photos from my cellphone to my data collection laptop which has no wifi network to connect to and in remote locations.

**Comment by LoveMyLand** [comment]
Or at least via a USB cable.

**Reply by bliepp** [reply]
Wouldn't that be kinda pointless? You should already be able to move photos from your phone to your laptop via USB.
